mgmtsystem/rest_client.py

```diff
@@ -57,10 +57,11 @@ def get(self, entity_type, name=None, namespace=None):
         r = self.raw_get(path)
         return (r.status_code, r.json() if r.ok else None)
 
-    def get_json(self, path):
-        r = self.raw_get(path)
+    def get_json(self, path, headers={"Hawkular-Tenant": "hawkular"}):
```

Comment:
As get_json is useful to any, I think we should make it generic and not have a hawkular default